### PR TITLE
Adds openh264

### PR DESCRIPTION
**Pull request by damencho**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -28,18 +36,23 @@
...
                <Bundle-NativeCode>
                 darwin/libjng722.jnilib;osname=Mac OS X;processor=x86-64,
+                darwin/libjnffmpeg.jnilib;osname=Mac OS X;processor=x86-64,
+                darwin/libjnffmpeg-no-openh264.jnilib;osname=Mac OS X;processor=x86-64,
                 linux-x86/libjng722.so;osname=Linux;processor=x86,
                 linux-x86-64/libjng722.so;osname=Linux;processor=x86-64,
                 win32-x86/jng722.dll;osname=Win32;processor=x86,
-                win32-x86-64/jng722.dll;osname=Win32;processor=x86-64
+                win32-x86/jnffmpeg.dll;osname=Win32;processor=x86,
+                win32-x86-64/jng722.dll;osname=Win32;processor=x86-64,
```
Comment (ibauersachs) [inline pom.xml]: If all jng722 dlls are deleted, this must be reflected here too. But why were they deleted at all?

Comment (damencho) [reply pom.xml]: Current master has the binaries in two places. And this change just removes them from lib where they are not used.

File: pom.xml [hunk 2]
```diff
@@ -11,14 +11,22 @@
...
 
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+            <source>1.8</source>
+            <target>1.8</target>
+        </configuration>
```
Comment (damencho) [inline pom.xml]: Hey @ibauersachs why we added it as 1.8? Maybe we should fix this, we want current jitsi-desktop to be 1.7 dependent and from next release we will start with 1.8, right?

Comment (ibauersachs) [reply pom.xml]: Probably out of habit, can't remember. I don't mind which version we target for this specific release, but I probably used 1.8 features that would need to be reverted.

File: pom.xml (hunk 1 above)
Comment (ibauersachs) [reply pom.xml]: Ok, didn't notice that.

**Comment by ibauersachs** [comment]
I can't go through the entire changeset again. If it works, fine :-)

**Comment by bgrozev** [comment]
I can't go through the change set either :)
Looks fine to me, except for the language level thing. My understanding is that we are trying to keep the next release of Jitsi at 1.7 (which is why we've holding libjitsi master at 1.7). But I'd be happy either way

**Pull request merged**